tests/Entity/SnapshotManagerTest.php

```diff
@@ -226,7 +226,11 @@ public function testEnableSnapshots(): void
         $page = $this->createMock(PageInterface::class);
         $page->expects(static::once())->method('getId')->willReturn(456);
 
-        $snapshot = $this->createMock(Snapshot::class);
+        $snapshot = $this->getMockBuilder(SnapshotInterface::class)
+            ->disableOriginalConstructor()
+            ->addMethods(['getId'])
+            ->getMockForAbstractClass();
+        // @phpstan-ignore-next-line
```

Comment:
If we need this method, we should add it to the interface.

Reply:
Not possible on the stable branch...

Reply:
At least `@method` should be added https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Model/SnapshotInterface.php#L28
And a next major can be added in the test to update the code in the test.

But in this special case, I'm not sure we can consider that adding `getId` to the interface is a BC-break.
It's called here: https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Entity/SnapshotManager.php#L91, so the method must be implemented anyway.

Reply:
done

Reply:
the phpstan ignore is not needed anymore i think